### ISSUE TITLE
Is this project dead?

### ISSUE THREAD

**Issue by Minolta1034** [bradp/vv#360]
Hasn't been a release in almost a year and it seems like it does not work with VVV 2.0? 

Is this project still being worked on or is there a fork of it that is still active?

**Comment by kcwoodfield** [comment]
Same issue here.  VVV 2.0 is giving me issues.

**Comment by neilgee** [comment]
I am working with it in VVV2 - as long as you have added a sites details in your VVV directories file  `vvv-custom.yml` - if you don't have that file make a copy and rename it from `vvv-config.yml`

(With Chrome and .dev domains being unusable - make sure you use .test as the domain suffix)

Add the site like so copying the same style spacing as existing sites...

```
  example:
    hosts:
      - example.test
```

Then use `vv` the normal way

**Comment by PorterBytes** [comment]
@neilgee I'm doing this right now (been trying for a half hour, with a fresh download of vagrant included), and this simply isn't working. I'm getting an "example" directory, with the hosts, init, and cli files, but no htdocs folder. 

Any idea what's causing this? I'm stumped, and working is kind of halted until I figure it out.

**Comment by nelga** [comment]
I found this yesterday, planning to check it out. Might be a good alternative full-stop, or at least until they update VV. https://github.com/JPry/vvv-base

**Comment by tarikhamilton** [comment]
That's the vibe I'm getting. I'm not able to run shell scripts inside the VM, so I can't use the wonderful automation script at all anymore. Not sure if that issue is from VVV 2 or VV.

**Comment by kcwoodfield** [comment]
@tikitariki, @nelga I went and grabbed the 1.4.1 release of VVV and i'm back where I was before I updated to VVV 2.0.  1.4.1 was working great for me and VVV 2.0 (which I'm sure is fine) isn't supported yet. 

https://github.com/Varying-Vagrant-Vagrants/VVV/tree/1.4.1

**Comment by hobbsy** [comment]
how do I downgrade from 2.0 to 1.4.1?

**Comment by kcwoodfield** [comment]
@hobbsy download and install this release: https://github.com/Varying-Vagrant-Vagrants/VVV/tree/1.4.1

**Comment by hobbsy** [comment]
@kcwoodfield thanks. Do I need to uninstall v2.0 first? Will this delete any databases/sites already in place?

**Comment by kcwoodfield** [comment]
I have both VVV2 and VVV running it does not play nice so I turn off (`vagrant halt`) the one I'm not using.  I'm slowly migrating over all my sites to VVV2.  Most importantly because of the .dev TLD issue. 

As for the database thing, I'd say no since they exists in the virtual env.  Won't delete any sites either. 

I created a .../Sites/VVV2... directory and clone VVV2 into that one.

**Comment by ThatKDB** [comment]
VV seems to be dead. Even if it works under VVV 2, it writes everything directly in the nginx.conf and not in vvv-custom.yml. I've been using the [vvv-base repo](https://github.com/JPry/vvv-base), which while not quite as simple as vv, is quite easy.

**Comment by kcwoodfield** [comment]
@ThatKDB yeah, I used VV (was awesome) with VVV (and even this nice dashboard https://github.com/topdown/VVV-Dashboard) but I've totally abandoned VVV v1.x for VVV2.  

Creating new sites isn't so bad at all actually and the .test domain puts me at ease knowing .dev doesn't work anymore. 

I highly suggest going with VVV2 and just following their docs.

**Comment by PorterBytes** [comment]
Can confirm, vvv-base is a solid alternative. Just a quick copy and paste into your vvv-custom.yml, with a site name change.
